orchestrator/examples/run_benchmark_auto.sh

```
@@ -0,0 +1,199 @@
+#   --console-output BOOL     Print results to console (default: true)
+#   --file-output BOOL        Save results to file (default: true)
+#   --committee N             Committee size (default: 4)
+#   --faults N               Number of faulty nodes (default: 0)
+#   --crash-recovery BOOL    Crash recovery enabled (default: false)
+#   --crash-interval N       Crash interval in seconds (default: 60)
+#   --duration N             Benchmark duration in seconds (default: 180)
+#   --network-type TYPE      Network type to benchmark (local or remote) (default: local)
+#   --transaction-size SIZE  Transaction size in bytes (default: 512)
+#   --help                   Show this help message
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
```

Shellcheck (0.10.0)

[warning] DIR appears unused. Verify use (or export if used externally).

(SC2034)